style.css

```diff
@@ -127,3 +127,668 @@ li {
+  margin-top: -374px;
+}
+
+.support2 {
+  position: static;
+  width: 295px;
+  height: 87px;
+  left: 16px;
+  top: 101px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: -270px;
+  margin-left: -15px;
+}
+
+.tagB {
+  font-family: 'Inter', sans-serif;
+  position: static;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-style: normal;
+  line-height: 20px;
+  margin-left: -9px;
+  margin-top: -250px;
+  float: left;
+  color: #fff;
+}
+
+#tag5 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 55px;
+  height: 40px;
+  left: 1px;
+  top: 1px;
+  background-color: rgba(255, 255, 255, 0.24);
+  border: 1px transparent;
+  box-sizing: border-box;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 1px 12px;
+}
+
+#tag6 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 90px;
+  height: 40px;
+  left: 1px;
+  top: 1px;
+  background-color: rgba(255, 255, 255, 0.24);
+  border: 1px transparent;
+  box-sizing: border-box;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 1px 12px;
+}
+
+#tag7 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 61px;
+  height: 40px;
+  left: 1px;
+  top: 1px;
+  background-color: rgba(255, 255, 255, 0.24);
+  border: 1px transparent;
+  box-sizing: border-box;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 1px 12px;
+}
+
+.btn1 {
+  height: 48px;
+  width: 329px;
+  color: white;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  background-color: #ff6b00;
+  transition: solid 0s;
+  border-color: transparent;
+  margin-top: 129px;
+  margin-left: -300px;
+  position: static;
+}
+
+.tab {
+  margin-top: -2600px;
+  position: absolute;
+  width: 400px;
+  height: 2500px;
+  left: 1px;
+  top: 4137px;
+  background: #1c1a19;
+  background-repeat: no-repeat;
+  background-position: right;
+  background-image: url(illustration-about-me-mobile.png);
+  background-position-y: 400px;
+}
+
+.about {
+  text-align: center;
+  position: static;
+  width: 327px;
+  height: 79px;
+  left: 24px;
+  top: 120px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: -30px;
+  margin-top: 70px;
+}
+
+.support3 {
+  position: static;
+  width: 327px;
+  height: 148px;
+  left: 24px;
+  top: 223px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: -30px;
+}
+
+.btn3 {
```

Comment:
- [ ] All buttons should have their actions (such as hover and active) styled. You should consider adding them according to the provided design:

![image](https://user-images.githubusercontent.com/66967274/143685165-c6c981b1-5595-4e3c-b88a-6712cb4d131e.png)